project/src/main/ui/chat/chat-line-panel.gd

```diff
@@ -13,12 +13,7 @@ const PULSE_AMOUNT := Vector2(0.015, 0.030)
 const CHAT_TEXTURE_COUNT := 16
 
 # Longer lines of dialog are displayed in bigger panels
```

Comment:
This comment is slightly out of date, and should be updated

Reply:
Improved comment, `Different panel sizes to try, ordered from smallest to largest.`